### ISSUE TITLE
Check if install works on Rv4.0.0

### ISSUE THREAD

**Issue by ychae** [CRI-iAtlas/shiny-iatlas#197]
Using docker image `rocker/tidyverse:4.0.0-ubuntu18.04`

**Comment by ychae** [comment]
## iAtlas Rv4.0.0 installation test

Used the `rocker/tidyverse:4.0.0-ubuntu18.04` image to test installation.

```bash
docker run -d --name rv4installtest -p 8787:8787 -e ROOT=TRUE -e PASSWORD=PASS_ENV rocker/tidyverse:4.0.0-ubuntu18.04
```

Followed instructions in the shiny-iatlas github README to install 

Opening the `.Rproj` file, this warning appears:

```
Warning message:
Project requested R version '3.6.0' but '4.0.0' is currently being used 
```

Errors encountered when running `renv::restore()`:

1. `Error: failed to determine Bioconductor repositories`
   - Solved: updated the version of `Bioconductor` in the `renv.lock` file to ` "Version": "3.11"` 
2. `Error: install of package 'Cairo' failed`
   - Solved by: 
   - `sudo apt-get update; sudo apt-get install -y libgtk2.0-dev xvfb libxt-dev`
   - updated the version of `Cairo` in the `rev.lock` file to `"Version": "1.5-12"`

SUCCESS! 

TL;DR - can install successfully on R v4.0.0 by updating the `renv.lock` file entries for Bioconductor and Cairo

**Comment by Gibbsdavidl** [comment]
For installing packages, these changes fixed it for me.

When running it with R 4.0.2, I get this error:  

https://stackoverflow.com/questions/62340269/problems-with-dplyr-summarise-function-of-the-latest-r-version

**Comment by Gibbsdavidl** [comment]
Updating fixed it.
The following package(s) will be updated:

# CRAN ===============================
- dplyr   [0.8.3 -> 1.0.0]
